packages/sdks/output/vue/nuxt-isolated-vm-plugin.js

```diff
@@ -0,0 +1,12 @@
+export default defineNuxtPlugin((nuxtApp) => {
```

Comment:
- this works without needing to import `defineNuxtPlugin`?
- one last thing is to add an `exports` field for this plugin, so that customers can import it directly if they'd like. could be under `@builder.io/sdk-vue/nuxt-initialize-node-runtime-plugin`

Reply:
i haven't wrapped up this yet, will ping you once I test this out. got caught up with the Angular thing